corehq/doctypemigrations/djangomigrations.py

```diff
@@ -37,6 +38,7 @@ class MigrationNotComplete(Exception):
 
 def assert_initial_complete(migrator):
     def forwards(apps, schema_editor):
-        if not migrator.last_seq and not settings.UNIT_TESTING:
+        is_fresh_install = os.environ.get('CCHQ_IS_FRESH_INSTALL')
```

Comment:
i think we should also explicitly check for equality here:
`is_fresh_install = os.environ.get('CCHQ_IS_FRESH_INSTALL') == '1'`
on the off chance someone sets it to '0' and wouldn't want unexpected behavior